BLSQ/iaso
# POLIO-630 Fix timeout on calendar map merged shape- precalculate shape at update

Precalculate the geojson for each campaign at update time instead of when calling the API


## Self proof reading checklist

- [NA] Did I add translations
- [?] Are there enough tests

## Changes

- Added `geojson` GeoJson field  on Campaign with it's shape (well a list of)
- Also rigged the admin so it's recalculate when updating the model from there
- and added a Admin action to recalculate shapes
- Added a migration

## How to test

With polio setup
Try to change the scope a campaign, multiple time
See if it is still displayed in the Calendar
Also in Anonymous/embedded mode
